### ISSUE TITLE
Cursor is flickering at the edge of tooltip

### ISSUE THREAD

**Issue by danielpencak** [reactstrap/reactstrap#1728]
- components: `Tooltip`
- reactstrap version `#8.1.1`
- react version `#16.12.0`
- bootstrap version `#4.4.1`
- macOS `#10.15.1`
- Chrome `#Version 78.0.3904.108 (Official Build) (64-bit)`

### What is happening?

When the cursor, in default state, intercepts the boundary of a tooltip trigger and is left at that position the cursor flickers and transitions between the default and pointer states. In this position the element that the tooltip is attached to cannot be clicked or if it can be clicked it takes multiple clicks.

This issue is currently reproducible on the tooltip page of the reactstrap site.

[Tootip Issue](https://vimeo.com/user60206101/review/378344360/91ccd40be2)

### What should be happening?

Once the cursor intercepts the boundary of the element that the tooltip is attached to the cursor should be in the pointer state and the element that the tooltip is attached to, should be able to be interacted with.

**Comment by jansensan** [comment]
Currently unable to reach the video linked by @danielpencak, but we are able to reproduce this issue in our project:

![reactstrap-tooltip-issue](https://user-images.githubusercontent.com/1410355/71188425-f3b8f600-224e-11ea-87dc-5bce22f5150f.gif)

- components: `UncontrolledTooltip`
- reactstrap version `#8.1.1`
- react version `#16.12.0`
- bootstrap version `#4.3.1`
- macOS `#10.14.6`
- Chrome `Version 79.0.3945.88 (Official Build) (64-bit)`

This doesn't occur on Safari (`Version 13.0.3 (14608.3.10.10.1)`) or Firefox (`71.0 (64-bit)`)

**Comment by TheSharpieOne** [comment]
it seems like the tooltip is added to the DOM (faded out, getting read to fade in) at the point of the cursor. This causes the cursor to be over the tooltip itself and not the trigger, which causes the tooltip to fade out and be removed from the DOM, which then puts the cursor back over the trigger... rinse and repeat.

**Comment by jansensan** [comment]
@TheSharpieOne maybe the tooltip element itself could have `pointer-events: none` in its styles? Is there a case where there needs to be mouse interactions/taps with a tooltip?

**Comment by julesair** [comment]
I experience a similar problem on Chrome, but not on Firefox. It is definitely related to the tooltip component, because the flicker does not appear, when I remove it. Setting `pointer-events: none` on the little green pill disables triggering the tooltip. Setting `pointer-events: none` on the tooltip does not change the flickering.
![Peek 2019-12-20 14-15](https://user-images.githubusercontent.com/11719541/71258196-dca10380-2335-11ea-9a4a-01fe4ac93501.gif)

Its interesting, that the flicker causes to change the hover state of the table row on and off.

**Comment by arublevsky** [comment]
Disabling pointer events on the tool-tip element can be used as a workaround
```css
.tooltip {
    pointer-events: none;
}
```

**Comment by TheSharpieOne** [comment]
Yes, it's a good workaround, but it will break the `autohide={false}` functionality.

**Comment by Nishanth11** [comment]
You can use Reactstrap version 7.1.0. We think this issue is in version 8.0.1.
We experienced the same issue as @danielpencak, we were able to resolve the same by downgrading the version to 7.1.0.

**Comment by naveensg** [comment]
@**jansensan**  just try with reactstrap version ^7.0.1.

**Comment by mjangir** [comment]
> @TheSharpieOne maybe the tooltip element itself could have `pointer-events: none` in its styles? Is there a case where there needs to be mouse interactions/taps with a tooltip?

Yes if we want to copy the text of a tooltip. In that case autohide can be made false.

**Comment by alamothe** [comment]
Experiencing the same problem with `8.4.1` and Chrome.

`pointer-events: none;` works!
